.github/workflows/release-sigstore-gradle-plugin-from-tag.yaml

```diff
@@ -97,15 +97,15 @@ jobs:
     steps:
       - name: Download attestation
         # keep at v3.x since slsa generator uses 3.x (update this when slsa-framework updates)
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: actions/download-artifact@eaceaf801fd36c7dee90939fad912460b18a1ffe # v4.1.2
```

Comment:
this should not be updated

Reply:
not really aware of any renovate bot config to ignore this config though :\

Reply:
What do you want to ignore?

Reply:
those download-artifact actions that are at v3.0.2 are for downloading artifacts uploaded by a v3 action deep in the slsa generator. (they should update, but we're waiting on that here: https://github.com/slsa-framework/slsa-github-generator/issues/3068)